custom_components/toyota/manifest.json

```diff
@@ -7,6 +7,6 @@
   "documentation": "https://github.com/DurgNomis-drol/ha_toyota",
   "iot_class": "cloud_polling",
   "issue_tracker": "https://github.com/DurgNomis-drol/ha_toyota/issues",
-  "requirements": ["mytoyota==2.1.1", "arrow"],
+  "requirements": ["mytoyota==2.1.4", "arrow"],
```

Comment:
We should use a range to allow all non major releases of the mytoyota lib. This will give us more freedom for upcoming releases. Please have a look at my Pull Requests here: https://github.com/DurgNomis-drol/ha_toyota/pull/257